blog/.vuepress/plugins/sidebar/create.js

```diff
@@ -14,19 +14,27 @@ module.exports = filesExtractedDatas => {
     }
   });
 
-  const navBarAddition = {
-    text: "Archives",
-    link: "/archives/" + sidebar[0][0]
-  };
-
   const Events = {
     text: "Events",
-    link: "/events/index.html"
-  };
-
-  const Tags = {
-    text: "Tags",
-    link: "/tags/index.html"
+    items: [
+      {
+        text: "Group 1",
+        items: [
```

Comment:
@f3ltron I'm new to vuepress, is there a way to add a map file to the prod build to have know exactly where it fails?
I'm able to reproduce locally.
Looks like `items: []` here is the culprint, but I don't know why.

Reply:
i am not sure to understand what mean by a map file

Reply:
I meant the JS source map file.

I found that adding them mysteriously fixes the issue in build mode:
in `blog/.vuepress/config.js`:
```
configureWebpack: (config, isServer) => {
  if (!isServer) {
    return {
      devtool: 'source-map',
    }
  }
}
```

Reply:
It's look it's a vuepress bug so hum.. I will have a workaround for just having the result i want and i will post an issue for that so..

Reply:
yeah this is definitely coming from the enhanceapp logic... i dont really know why for now